tests/sentinel/tests/00-base.tcl

```diff
@@ -200,7 +200,7 @@ test "SENTINEL RESET can resets the primary" {
     for {set j 0} {$j < 10} {incr j} {
         assert_equal 1 [S 0 SENTINEL RESET mymaster]
         set res1 [llength [S 0 SENTINEL SENTINELS mymaster]]
-        set res2 [llength [S 0 SENTINEL SLAVES mymaster]]
```

Comment:
lets keep it, i believe i add this for the old name coverage

Reply:
got it, I will update it and add the comment for old name coverage.